ureport/utils/__init__.py

```diff
@@ -485,7 +487,9 @@ def get_ureporters_locations_stats(org, segment):
         boundary_top_level = 0 if org.get_config('is_global') else 1
         boundaries = Boundary.objects.filter(org=org, level=boundary_top_level).values('osm_id', 'name').order_by('osm_id')
         location_counts = {k[6:]: v for k, v in org_contacts_counts.iteritems() if k.startswith('state')}
-
+    elif field_type == 'ward':
+        boundaries = Boundary.objects.filter(org=org, level=3, parent__osm_id__iexact=parent).values('osm_id', 'name').order_by('osm_id')
```

Comment:
once again, let's refactor these to a reusable constant or ENUMS, etc. we just sprinkle magic numbers all over the code.